static/app/utils/replays/replayReader.tsx

```diff
@@ -419,6 +420,18 @@ export default class ReplayReader {
 
   getLPCFrames = memoize(() => this._sortedSpanFrames.filter(isLCPFrame));
 
+  getVideoAttachments = memoize(() =>
+    (
+      this._sortedRRWebEvents.filter(
+        event => event.type === EventType.Custom && event.data.tag === 'video'
+      ) as VideoFrameEvent[]
+    ).map(event => ({
+      duration: event.data.payload.duration,
+      id: event.data.payload.segmentId,
+      timestamp: event.timestamp,
+    }))
+  );
```

Comment:
NIT: i'd put this kind of stuff into `hydrateFrames.tsx` and then endup passing in something like `videoFrames` next to the existing `rrwebFrames` array.

followup kind of thing, but it should make `isVideoReplay` slightly faster for web-based but long replays.

Reply:
Updated https://github.com/getsentry/sentry/pull/66111/commits/ddfea3440e0d1c4db61e79a8f54eff2aed9f8fbf